src/components/Button.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import "./Button.css";
+
+function Button(props) {
+  return (
+    <button className="button" onClick={props.onClick} onMouseEnter={props}>
```

Comment:
`onMouseEnter={props}` what are you using this for?

Reply:
I guess nothing 😞